ui/app/ducks/app/app.js

```diff
@@ -756,6 +758,21 @@ function reduceApp (state, action) {
         loadingMethodData: false,
       })
 
+    case actions.SHOW_SEED_PHRASE_BACKUP_AFTER_ONBOARDING:
```

Comment:
These two actions look like they may have been added accidentally in resolving a merge conflict.

Reply:
Hmm.... actually, `HIDE_SEED_PHRASE_BACKUP_AFTER_ONBOARDING` does appear to be used, as I just discovered.

`SHOW_SEED_PHRASE_BACKUP_AFTER_ONBOARDING` is still unused though.

Reply:
No, you were right, both were accidentally re-added with a merge conflict. Here is where they were deliberately removed on a PR that was merged to develop: https://github.com/MetaMask/metamask-extension/pull/6961/commits/58f90d734ba885a2dbdd14f3c0761d0951c6cfd5

`HIDE_SEED_PHRASE_BACKUP_AFTER_ONBOARDING` is dispatched by a method that is never actually called. I will deal with that dead code in a separate PR.